x/programs/runtime/import_state.go

```diff
@@ -57,14 +57,14 @@ func NewStateAccessModule() *ImportModule {
 				}
 				return val, nil
 			})},
-			"put": {FuelCost: writeCost, Function: FunctionWithOutput(func(callInfo *CallInfo, input []byte) ([]byte, error) {
+			"put": {FuelCost: writeCost, Function: FunctionNoOutput(func(callInfo *CallInfo, input []byte) error {
```

Comment:
@dboehm-avalabs, this makes sense, right? 

As I ask the question, I'm realizing that we should probably be consistent between `delete` and `put`. Either both should return the old value like the Rust `HashMap` API, or both should return nothing.

As much as I like the Rust API, I'm in favour of the latter since it could be expensive to copy all the bytes into the wasm module

Reply:
I'd probably lean return nothing.

Reply:
I'll change that in a follow up